python/cugraph/structure/number_map.py

```diff
@@ -10,100 +11,45 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
 
-import cudf
+from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_distributed_data
+from cugraph.structure import renumber_wrapper as c_renumber
+import cugraph.comms.comms as Comms
 import dask_cudf
 import numpy as np
-import bisect
+import cudf
+
+
+def call_renumber(sID,
+                  data,
+                  num_edges,
+                  is_mnmg,
```

Comment:
In case of single node multi-gpu, shouldn't this be false? If this should be true in single-node multi-gpu, isn't this a misnomer?